src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2026,13 +2034,17 @@ private IOperation CreateBoundDiscardPatternOperation(BoundDiscardPattern boundN
         private IOperation CreateUsingLocalDeclarationsOperation(BoundUsingLocalDeclarations boundNode)
         {
             //TODO: Implement UsingLocalDeclaration operations correctly.
-            //      For now we return an implicit operationNone, with a single child consisting of the using declaration parsed as if it were a standard variable declaration
+            //      For now we return an implicit operationNone, 
+            //      and GetIoperationChildren will return a single child 
```

Comment:
>o [](start = 28, length = 1)

Nit: spelling

Reply:
Also trailing whitespace.

---
In reply to: [289972989](https://github.com/dotnet/roslyn/pull/36055#discussion_r289972989) [](ancestors = 289972989)

Reply:
fixed, and fixed, assuming I got the right spelling mistake :-)